src/test/java/seedu/address/model/AddressBookTest.java

```diff
@@ -83,6 +86,35 @@ public void getPersonList_modifyList_throwsUnsupportedOperationException() {
         assertThrows(UnsupportedOperationException.class, () -> addressBook.getPersonList().remove(0));
     }
 
+    @Test
+    public void sortPersons_emptyList_noChange() {
+        AddressBook emptyAddressBook = new AddressBook();
+        emptyAddressBook.sortPersons(Comparator.comparing(person -> person.getName().toString()));
+        assertEquals(0, emptyAddressBook.getPersonList().size());
+    }
+
+    @Test
+    public void sortPersons_singlePerson_noChange() {
+        AddressBook singlePersonBook = new AddressBook();
+        Person singlePerson = new PersonBuilder().withName("Single Person").build();
+        singlePersonBook.addPerson(singlePerson);
+
+        singlePersonBook.sortPersons(Comparator.comparing(person -> person.getName().toString()));
+        assertEquals(1, singlePersonBook.getPersonList().size());
+        assertEquals(singlePerson, singlePersonBook.getPersonList().get(0));
+    }
+
+    @Test
+    public void sortPersons_typicalList_sortedCorrectly() {
+        AddressBook addressBook = getTypicalAddressBook();
+
+        addressBook.sortPersons(Comparator.comparing(person -> person.getName().toString()));
+
+        assertEquals(ALICE, addressBook.getPersonList().get(0));
```

Comment:
It is helpful that individuals having first names starting with successive letters are used for testing.